accelerator/roles/intel/tasks/install_ubuntu.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Check if accelerator is present on node
+  ansible.builtin.include_tasks: verify_has_accelerators.yml
```

Comment:
is this separate playbook required just for a lspci command?

Reply:
we can integrate this one in this playbook if it's not needed somwhere else